pkg/deploy/internal/document/document.go

```diff
@@ -148,17 +148,22 @@ func createResolvedEntity(documentName string, id string, coordinate coordinate.
 	}
 }
 
-func getDocumentAttributesFromConfigType(t config.Type) (documents.DocumentType, bool, error) {
+var documentMapping = map[config.DocumentKind]documents.DocumentType{
```

Comment:
Does it make sense to centralize this in a helper and use it also in the `download.go`? Just with reverse mapping :)

Reply:
I thought about that as well, but could not find a good position. I don't know why the deploy/download/delete code is sooo split up :(

Any good idea where to put it? /internal/document package maybe somewhere?

Reply:
I understand I think `/internal/document/` sounds good.

Reply:
Lets move this to a refactoring as discussed in the call :)